test.py

```diff
@@ -31,7 +32,7 @@ def failing_call_with_output(cmd, expected):
   assert expected in stdout, 'call did not have the right output'
 
 def hack_emsdk(marker, replacement):
-  src = open('emsdk').read()
+  src = open('emsdk.py').read()
```

Comment:
why are the tests changed? none of the old usages are tested, so we lost coverage for the old `./emsdk` use pattern, which is what users are doing. best to test both (the official/intended, and also I assume some people will call the new `emsdk.py` directly because they find it).

Reply:
This test in particular is hacking the python source itself, so it needs to change.